public/src/components/channelManagement/choiceCards/ChoiceCardEditor.tsx

```diff
@@ -0,0 +1,311 @@
+                  </RadioGroup>
+                )}
+              />
+            </FormControl>
+          )}
+        </div>
+
+        <FormControlLabel
+          control={
+            <Controller
+              name={`choiceCards.${index}.isDefault`}
+              control={control}
+              render={({ field }) => (
+                <Checkbox
+                  {...field}
+                  checked={field.value}
+                  color="primary"
+                  disabled={isDisabled}
+                  onChange={e => {
+                    field.onChange(e.target.checked);
+                    handleCardChange();
+                  }}
+                />
+              )}
+            />
+          }
+          label="Is Default"
+        />
+
+        {getValues(`choiceCards.${index}.product.supportTier`) === 'OneOff' && (
+          <Controller
+            name={`choiceCards.${index}.label`}
+            control={control}
+            rules={{
+              required: EMPTY_ERROR_HELPER_TEXT,
+            }}
+            render={({ field, fieldState }) => (
+              <RichTextEditorSingleLine
+                copyData={field.value}
+                updateCopy={value => {
+                  field.onChange(value);
+                  handleCardChange();
+                }}
+                name={`label-${index}`}
+                label="Label"
+                disabled={isDisabled}
+                error={!!fieldState.error}
+                helperText={fieldState.error?.message}
+                rteMenuConstraints={richTextEditorConfig}
+              />
+            )}
+          />
+        )}
+
+        <Controller
+          name={`choiceCards.${index}.pill.copy`}
+          control={control}
+          render={({ field }) => (
+            <TextField
+              {...field}
+              required={false}
+              label="Pill Copy (optional)"
+              variant="filled"
+              fullWidth
+              margin="normal"
+              disabled={isDisabled}
+              onChange={e => {
+                field.onChange(e);
+                handleCardChange();
+              }}
+            />
+          )}
+        />
+
+        <Controller
+          name={`choiceCards.${index}.benefitsLabel`}
+          control={control}
+          render={({ field }) => (
+            <RichTextEditorSingleLine
+              copyData={choiceCard.benefitsLabel || ''}
+              updateCopy={value => {
+                field.onChange(value);
+                handleCardChange();
+              }}
+              name={`benefitsLabel-${index}`}
+              label="Benefits Label (optional)"
+              disabled={isDisabled}
+              error={false}
+              rteMenuConstraints={richTextEditorConfig}
+            />
+          )}
+        />
+
+        <div>
+          <label className={classes.benefitsHeading}>Benefits</label>
```

Comment:
I notice the benefit added to the recurring contribution in the test is not showing in the CHOICE_CARDS_CONFIG:CONTROL epic - is DCR is updated to pick up the benefits in the configuration yet?